test/built-ins/Realm/prototype/evaluate/wrapped-functions-new-wrapping-on-each-evaluation.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2021 Rick Waldron. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-realm.prototype.evaluate
+description: >
+  Realm.prototype.evaluate wrapped functions produce new wrapping on each evaluation.
+features: [callable-boundary-realms]
+---*/
+
+assert.sameValue(
+  typeof Realm.prototype.evaluate,
+  'function',
+  'This test must fail if Realm.prototype.evaluate is not a function'
+);
+
+const r = new Realm();
+
+r.evaluate(`
+function fn() {
+    return 42;
+}
```

Comment:
oops, evaluate will return undefined here

Reply:
It doesn't need to be, since the test is evaluating `fn` later and getting the wrapped function there.